### ISSUE TITLE
The “/esm” directory cannot load via a “module” import.

### ISSUE THREAD

**Issue by theengineear** [linkeddata/rdflib.js#657]
👋 — Hi there! I was hoping to be able to leverage `rdflib` to visualize some modeling problems I’m working on in a browser. However, when I went to get started, I couldn’t get it loading via the exported `/esm` scripts. E.g., this doesn’t work:

```html
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <title>rdflib demo</title>
    <script type="module" src="https://www.unpkg.com/rdflib@2.2.35/esm/index.js"></script>
  </head>
  <body></body>
</html>
```

This will complain about `@babel` not existing, but I don’t believe an import map declaration to map the `@babel` bare module specifier to a well-defined resolution destination will work… If you look in `https://www.unpkg.com/rdflib@2.2.35/esm/index.js`, you’ll see lines like:

```js
import _createClass from "@babel/runtime/helpers/createClass";
```

But `…createClass` isn’t a real thing. You can find `…createClass.js` (https://www.unpkg.com/@babel/runtime@7.25.7/helpers/createClass.js), but because there’s no `.js`, the browser tries to lookup (https://www.unpkg.com/@babel/runtime@7.25.7/helpers/createClass). And… that doesn’t exist.

In other words, this doesn’t really help to do the following:

```html
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <title>rdflib demo</title>
    <script type="importmap">
      {
        "imports": {
          "@babel/runtime/": "https://www.unpkg.com/@babel/runtime@7.25.7/"
        }
      }
    </script>
    <script type="module" src="https://www.unpkg.com/rdflib@2.2.35/esm/index.js"></script>
  </head>
  <body></body>
</html>
```

❤️ — Thanks for taking a look!
